xblockmufi/xblockmufi.py

```diff
@@ -72,27 +71,56 @@ def workbench_scenarios():
     """
     Main functions
     """
-    def student_view(self, context=None):
+    
+    def build_fragment(
+             self,
+             template,
+             context_dict,
+             initialize_js_func,
+             additional_css=[],
+             additional_js=[],
```

Comment:
I know this convention predates you, but see the comment from Sahar's PR that I CC'ed you on,
re: defaulting to collections. These should take the form:

```py
def foo(additional_js=None):
    additional_js = additional_js or []
```